File: app/src/main/java/com/stevenschoen/putionew/model/responses/SubtitlesListResponse.java
package com.stevenschoen.putionew.model.responses;

import com.google.gson.annotations.SerializedName;
import com.stevenschoen.putionew.model.ResponseOrError;
import com.stevenschoen.putionew.model.files.PutioSubtitle;

import java.util.List;

public class SubtitlesListResponse extends ResponseOrError.BasePutioResponse {
	private List<PutioSubtitle> subtitles;
    @SerializedName("default")
    private String defaultSubtitleKey;

	public List<PutioSubtitle> getSubtitles() {
		return subtitles;
	}

    public String getDefaultSubtitleKey() {
        return defaultSubtitleKey;
    }
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/SubtitlesListResponse.kt
package com.stevenschoen.putionew.model.responses

import com.squareup.moshi.Json
import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.files.PutioSubtitle

class SubtitlesListResponse(
    val subtitles: List<PutioSubtitle>,
    @Json(name = "default")
    val defaultSubtitleKey: String
) : ResponseOrError.BasePutioResponse()
